app/Console/Commands/BulkIndex.php

```diff
@@ -277,8 +276,8 @@ protected function getContracts()
         $query    = $this->contract;
 
         if ($category != 'all') {
-            $from = "contracts ";
-            $from .= ",json_array_elements(contracts.metadata->'category') cat";
+            $from  = "contracts ";
+            $from  .= ",json_array_elements(contracts.metadata->'category') cat";
```

Comment:
don't use an abbreviation in SQL command; try to make it self descriptive as much you can